dyno-core/src/main/java/com/netflix/dyno/connectionpool/impl/lb/HttpEndpointBasedTokenMapSupplier.java

```diff
@@ -44,26 +44,26 @@
     private static final String DefaultServerUrl = "http://{hostname}:{port}/REST/v1/admin/cluster_describe";
     private static final Integer NUM_RETRIES_PER_NODE = 2;
     private static final Integer NUM_RETRIER_ACROSS_NODES = 2;
-    private static final Integer defaultPort = 8080;
+    private static final Integer defaultHttpPort = 8080;
 
     private final String serverUrl;
 
-    public HttpEndpointBasedTokenMapSupplier() {
-        this(DefaultServerUrl, defaultPort);
+    public HttpEndpointBasedTokenMapSupplier(int dynomitePort) {
+        this(DefaultServerUrl, defaultHttpPort, dynomitePort);
     }
     
-    public HttpEndpointBasedTokenMapSupplier(int port) {
-	this(DefaultServerUrl, port);
+    public HttpEndpointBasedTokenMapSupplier(int httpPort, int dynomitePort) {
+	this(DefaultServerUrl, httpPort, dynomitePort);
     }
 
-    public HttpEndpointBasedTokenMapSupplier(String url, int port) {
-	super(port);
+    public HttpEndpointBasedTokenMapSupplier(String url, int httpPort, int dynomitePort) {
+	super(dynomitePort);
```

Comment:
Are you sure that super should not be `super(httpPort)`? Was it a bug before?

Reply:
the port is not being used and it was passed unnecessary because http port should have no role in AbstractTokenMapSupplier. Http port is really a property of HttpEndPointBasedTokenMapSupplier.